pkg/operator/hive/hiveadmission.go

```diff
@@ -329,18 +329,33 @@ var allowedContracts = sets.NewString(
 	hivecontractsv1alpha1.ClusterInstallContractLabelKey,
 )
 
+// knowContracts is a list of contracts and their implementations that doesn't
+// require discovery using CRDs
+var knowContracts = contracts.SupportedContractImplementationsList{{
+	Name: hivecontractsv1alpha1.ClusterInstallContractName,
+	Supported: []contracts.ContractImplementation{{
+		Group:   "extensions.hive.openshift.io",
+		Version: "v1",
```

Comment:
AgentClusterInstal (and all other assistedservice apis) are v1beta1 in their repo right now.

@rollandf @mhrivnak do we want to keep that or move it to v1 for AgentClusterInstall?

Reply:
If we can keep it at v1beta1, that's better for now since it's brand new. We can use whatever flexibility we can get.

Reply:
updated to v1beta1